examples/command_center/command_center.ino

```diff
@@ -4,101 +4,63 @@
 // This example code shows how to use an Adafruit Feather M0 module and Microsoft Azure for IoT
 // applications.
 
-#include <stdlib.h>
-#include <stdio.h>
-#include <stdint.h>
-#include <time.h>
-#include <sys/time.h>
-#include <SPI.h>
-#ifdef ARDUINO_SAMD_FEATHER_M0
+// change the next three lines to use on non-Adafruit WINC1500 based boards/shields
```

Comment:
What do you think about leaving in/combining the #ifdef logic? That way the sample can work as is for all boards without having to manually edit the includes?